app/views/foreman_ansible/job_templates/package_action_-_ansible_default.erb

```diff
@@ -41,7 +40,7 @@ model: JobTemplate
   <%- end -%>
   tasks:
     - package:
-        name: <%= input('name') %>
+<%= indent(8) { to_yaml({"name" => input('name')}).gsub(/---\n/, '') } -%>
```

Comment:
I was able to remove my erb loop/if pasta code. `to_yaml` macro doesn't seems to support `:UseHeader => true` https://github.com/ruby/ruby/blob/4099b04c099899d92dab3f34f1152038ed718d79/test/psych/helper.rb#L51